src/GUI/GamePanel.java

```diff
@@ -0,0 +1,79 @@
+package GUI;
+
+import Gameplay.*;
+
+import javax.swing.*;
+import java.awt.*;
+import java.io.IOException;
+
+public class GamePanel extends JPanel {
+
+    protected static int XOffset = 100;
+    public static Point deckPosition = new Point(620, 20);
+    public static Point tableauPosition = new Point(20, 150);
+    public static int tableauOffset = 100;
+    private static Deck deck;
+    private static WastePile wastePile;
+    private static Foundations[] foundations;
+    private static Tableau[] tableaus;
+    private ShuffleButton shuffleButton;
+
+    public GamePanel() throws IOException {
+        super.setLayout(null);
+        initializePiles();
+
+        GameMoveListener gameMoveListener = new GameMoveListener();
+        addMouseListener(gameMoveListener);
+        addMouseMotionListener(gameMoveListener);
+    }
+
+
+    private void initializePiles() throws IOException {
+            deck = new Deck(deckPosition.x, deckPosition.y);
+            add(deck);
+
+            wastePile = new WastePile(deckPosition.x - XOffset, deckPosition.y);
+            add(wastePile);
+
+            foundations = new Foundations[4];
+
+            for (int i = 0; i < foundations.length; i++) {
+                foundations[i] = new Foundations(20 + XOffset*i, 20, i + 1);
+                add(foundations[i]);
+            }
+
+            tableaus = new Tableau[7];
+            for (int i =0; i < tableaus.length; i++){
+                tableaus[i] = new Tableau(tableauPosition.x + tableauOffset * (i),
+                        tableauPosition.y, i +1);
+                add(tableaus[i]);
+            }
+
+            shuffleButton = new ShuffleButton(600,20, 100);
+            shuffleButton.setBounds(700, 20, 149, 49);
```

Comment:
I did elaborate on arbitrary constants a bit above. General term is "magic constant". 
If you computed these by hand instead of something like `windowWidth / 2 + shuffleXOffset, 0 /* default top Y */ + shuffleYOffset` (or `pileX + offset, pileY`), reconsider. 
Changing other things might become inconvenient due to this.